anatomy/tracks/concepts.md

```diff
@@ -41,45 +41,73 @@ concepts
 
-**Purpose:** Provide information about the concept for a student who has completed the corresponding concept exercise to learn from and refer back to.
+**Purpose:** Provide more detailed information about the concept for a student who has completed the corresponding concept exercise to learn from and refer back to.
 
 **Presence:** Required
 
-Once the student completes the exercise they will be shown this file, which should provide them with a summary of the concept. If the concept introduces new syntax, syntax samples should be included. At a minimum, this file should contain all information that is introduced in the exercise's [`.docs/introduction.md` document](./concept-exercises#filedocsintroductionmd).
+After unlocking the concept by completing the corresponding exercise, the student will be able to navigate to this file. It should provide them with comprehensive information on what they need to know to be fluent in the concept. At a minimum, this file should contain all information that is introduced in the concepts' [`introduction.md` document](./concepts.md#file-introductionmd).
 
-This document can also link to any additional resources that might be interesting to the student in the context of the exercise, such as:
+If the concept introduces new syntax, syntax samples should be included. The student should not have to follow a lot of links to gain the knowledge that the file tries to convey. Instead about.md should contain enough information to be understandable within its content.
+
+About.md is not limited to the scope of the corresponding concept exercise. The content can require knowledge of other concepts that will be introduced later on. If other concepts are mentioned, their respective introductions should be linked (how?).
+
+Here some examples of what could be covered.
 
 - Popular usages for a feature
 - Common pitfalls in a feature's use (e.g. casual use of multiple **threads**)
 - Limitations on use that may catch out the unsuspecting developer
-- Alternative approaches addressed in other exercises
+- Alternative approaches addressed in other concepts
 - Compromises made for ease of learning or to accommodate the Exercism environment, e.g. multiple classes in single file
 - Similar features with which the concept may be confused
 - Performance characteristics and memory usage
 
+It is NOT the aim of the about.md file to provide a complete set of information on the concept. As an example, imagine a language that has some older features for which experienced programmers (and maybe even the official docs/specs) recommend they should not be used anymore. Providing details on such features would out of scope for the about.md file because they are not relevant to gain fluency.
+
+Make sure the about.md file is clearly structured, especially when it contains a lot of information. In the future there will also be support for marking parts as "advanced topics" to point them out to interested students without overloading others.
```

Comment:
```suggestion

The `about.md` file MUST be clearly structured, especially when it contains a lot of information. In the future there will also be support for marking parts as "advanced topics" to point them out to interested students without overloading others.
```

Reply:
I use "MUST" here in the context of [RFC 2119](https://tools.ietf.org/html/rfc2119), which we use throughout these docs.